activities/recruitment/hiring-process.md

```diff
@@ -6,35 +6,36 @@ type: Resource
-4. HRM notifies candidates (thank you or invitation for interview).
-5. First interview: a lead codebase steward and HRM, approximately 1 hr, partially on the basis of a pre-established list of desirable competences and other characteristics (i.e. fit for role, fit for organization (culture, growth, leadership, long/short term)). Half of the interview is spent presenting PublicCode.net, in the other half, the candidate has a chance to present themselves. Immediately after the interview both HRM and CEO score the candidate on the items on the list and determine if the candidate should progress to a second interview. HRM will contact the candidate to inform them.
-6. Second interview: CEO with HRM, also partially based on the pre-established list, and with attention to issues not discussed in the first interview. Approximately. 80-90% of the time is spent on questioning the candidate. Immediately after the interview HRM and CEO score the candidate, based on the list and other impressions gathered. HRM takes this input to CEO, and HRM and CEO determine if the candidate should receive an offer.
-7. CEO and HRM determination of the offer, including salary range and benefits.
-8. Third interview, hopefully a formality, of the preferred candidate with HRM and a BOD member. Decision on making an offer is made after third interview.
-9. HRM makes the offer to the candidate and negotiates, always in consultation with CEO and the director(s).
-10. Signing of contract and starting by the new team member, supervised by their hierarchical supervisor.
+1. Current job openings have no closing date and applications are considered on a rolling basis. When we find a suitable candidate, we hire and close the opening. This means applications are considered ‘by themselves’ and not ‘in competition’ with a certain set of other applications
+2. OC confirms receipt to candidates and does first round of evaluation
+3. OC brings possibly suitable candidates to the attention of CE and the role's future supervisor. These 2 make +/- judgement, without consulting each other. If judgement is not equal, a short discussion follows and judgement is made
+4. OC notifies candidates (thank you or invitation for interview)
+5. First interview: a lead codebase steward, or coordinator, depening on the role, and the OC, approximately 1 hr, partially on the basis of a pre-established list of desirable competences and other characteristics (i.e. fit for role, fit for organization (culture, growth, leadership, long/short term)). Half of the interview is spent presenting PublicCode.net, in the other half, the candidate has a chance to present themselves. Immediately after the interview the lead codebase steward and OC score the candidate on the items on the list and determine if the candidate should progress to a second interview. OC will contact the candidate to inform them
```

Comment:
I've always disliked this formulation: it makes it sound like it'll be an hour of duelling powerpoints, rather than a dialogue driven by mutual curiosity.

> Half of the interview is spent presenting PublicCode.net, in the other half, the candidate has a chance to present themselves.